@types/index.d.ts

```diff
@@ -551,9 +544,16 @@ declare module "opentok-react-native" {
   /**
    * https://github.com/opentok/opentok-react-native/blob/main/docs/OTSubscriber.md#custom-rendering-of-streams
    */
-  export class OTSubscriberView extends React.Component<OTSubscriberViewProps> {}
+  export class OTSubscriberView extends React.Component<OTSubscriberViewProps, unknown> {}
```

Comment:
same:
React.Component **requires** two type arguments: <Props, State>

I did not want to dive too deep in what should be declared for State; thus `unknown` is a good start.